README.md

```diff
@@ -15,4 +15,45 @@ $ conan config install https://github.com/includeos/conan_config.git
 ```
 
 ## Using multiple conan configurations
-To prevent overwriting any existing conan configuration the IncludeOS configuration can be tested by creating a custom conan home. To do this set the environment variable `CONAN_USER_HOME` to a path where a `.conan` folder will be created. 
+To prevent overwriting any existing conan configuration the IncludeOS
+configuration can be tested by creating a custom conan home.
+To do this set the environment variable `CONAN_USER_HOME` to a path where a
+`.conan` folder will be created.
+
+
+### Profiles in Support
+
+The profiles directory contains a list of profiles that we currently use in the
+development and testing of [IncludeOS](https://github.com/includeos/includeos).
+Below is a list of profiles with description.
+
+#### Build on MacOS
+
+** Clang 6.0 : **
```

Comment:
You missed the ** here. 
also is `clang-10` fit to go under the clang 6.0 header?